owtf/api/handlers/auth.py

```diff
@@ -216,3 +240,68 @@ def get(self):
             self.success(data)
         else:
             raise APIError(400, "Invalid Token")
+
+
+class AccountActivationGenerateHandler(APIRequestHandler):
+    SUPPORTED_METHODS = ["POST"]
+
+    def post(self):
+        email_to = self.get_argument("email", None)
+        email_confirmation_dict = {}
+        email_confirmation_dict["key_value"] = str(uuid4())
+        email_confirmation_dict["expiration_time"] = datetime.now() + timedelta(hours=1)
+        user_obj = User.find_by_email(self.session, email_to)
+        email_confirmation_dict["user_id"] = user_obj.id
+        EmailConfirmation.remove_previous_all(self.session, user_obj.id)
+        EmailConfirmation.add_confirm_password(self.session, email_confirmation_dict)
+
+        html = (
+            """\
+        <html>
+        <body>
+            Welcome """
+            + user_obj.name
+            + ", <br/><br/>"
+            """ 
+            Click <a href=\""""
+            + "http://0.0.0.0:8009/email-verify/"
```

Comment:
you might want to get the host from settings because someone might be running this on 127.0.0.1

Reply:
Modified.